### PR TITLE
Remove outdated schedule, update text.

### PR DESCRIPTION
**Pull request by RoAnnC1**

### CODE REVIEW

File: src/tools-support/support.markdown [hunk 1]
```diff
@@ -11,29 +11,14 @@ redirect_from: /docs-and-resources/support/
...
-* April 22
-* May 20
-* June 24
-* July 22
-* August 19
-* September 16
-* October 14
-* November 4
-* December 9
+The SAP Concur platform undergoes routine maintenance each weekend, and releases updates monthly. During these periods, the service may be unavailable. Unplanned maintenance is performed rarely, and limited to high priority fixes. During downtime, any request to the SAP Concur Platform will receive an HTTP 503 error code. To check the current status see: http://open.concur.com/, a message will be posted regarding planned maintenance.
```
Comment (sarraloew) [inline src/tools-support/support.markdown]: Let's say "a message will be posting during planned maintenance"

File: src/tools-support/support.markdown [hunk 2]
```diff
@@ -11,29 +11,14 @@ redirect_from: /docs-and-resources/support/
 * [Partners](/tools-support/requesting-partner-support.html)
```
Comment (sarraloew) [inline src/tools-support/support.markdown:11]: On line 7, remove the North America and EMEA text- it's now for all data centers.

**Review by sarraloew — CHANGES_REQUESTED** [review]

File: src/tools-support/support.markdown (hunk 1 above)
Comment (RoAnnC1) [reply src/tools-support/support.markdown]: Done

File: src/tools-support/support.markdown (hunk 2 above)
Comment (RoAnnC1) [reply src/tools-support/support.markdown:11]: Done

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Pull request merged**